From 9711665bc0a552d2fcf9b236c3b17e8a9e7c4958 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Olivier=20Cr=C3=AAte?= <olivier.crete@ocrete.ca>
Date: Tue, 17 Jul 2018 12:24:03 -0400
Subject: [PATCH] cmake: Try winpthread before pthreadGC2 (#429)

It seems that pthreadGC2 is deprecated and it should prefer
winpthread. Also remove all of the commented out stuff.
---
 CMakeLists.txt | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3f8af98..27a1a5b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -338,15 +338,7 @@ if (${ENABLE_PROFILE} AND HAVE_COMPILER_GNU_COMPAT)
 endif()
 
 if (MINGW)
-	set (PTHREAD_LIBRARY -lpthreadGC2)
-	# XXX CONSIDER THIS. It came from a merged previous master version
-	# and was under an if(NOT MINGW)...elseif(THREADS_FOUND) condition.
-	# Might be that the current form still suffices; resolve this and remove comment.
-	#if(THREADS_FOUND)
-	#set(PTHREAD_LIBRARY ${CMAKE_THREAD_LIBS_INIT})
-	#else()
 	find_library(PTHREAD_LIBRARY NAMES pthread pthreadGC2 pthreadGC)
-	#endif() 
 elseif (PTHREAD_LIBRARY AND PTHREAD_INCLUDE_DIR)
 	message(STATUS "Pthread library: ${PTHREAD_LIBRARY}")
 	message(STATUS "Pthread include dir: ${PTHREAD_INCLUDE_DIR}")
